.yamato/yamato-config.yml

```diff
@@ -1,7 +1,7 @@
 name: Endpoint Unit Tests
 agent:
     type: Unity::VM
-    image: robotics/ci-ubuntu20:latest
+    image: robotics/ci-ubuntu20:v0.1.0-795910
```

Comment:
what is the new version number `v0.1.0-795910`?  What is changed in the `ci-ubuntu20` image?

Reply:
all of our configs had originally pointed to `latest`, and it was suggested that we point to a specific tag that is known to work, only upgrading when we're certain a new image won't break anything. here's the related [slack thread](https://unity.slack.com/archives/C998PJJDD/p1628267077107500?thread_ts=1628124938.073100&cid=C998PJJDD)!

the other repo changes were made for URDF-Importer [here](https://github.com/Unity-Technologies/URDF-Importer/pull/107#pullrequestreview-724595328) and for ROS-TCP-Connector [here](https://github.com/Unity-Technologies/ROS-TCP-Connector/pull/154#issuecomment-894553639)

Reply:
Ah, gotcha! sounds good!